statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2894,6 +2894,76 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def get_rsquared(self, baseline="rwdrift", **kwargs):
+        # if self.model.k_endog > 1:
+        #     raise NotImplementedError(
+        #         "To be Implemented for Multivariate Model")
+        def sse_finite():
```

Comment:
No need to make this a function anymore - it's probably a bit cleaner to compute `sse` instead of delegating that to a function.

Reply:
sure